packages/alea-frontend/components/ProblemCard.tsx

```diff
@@ -0,0 +1,51 @@
+import { Box, Typography, RadioGroup, FormControlLabel, Radio, Button } from '@mui/material';
+import { Problem } from '@stex-react/api';
+
+const ProblemCard: React.FC<{
+  problem: Problem | null;
+  selectedOption: string | null;
+  onOptionChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+}> = ({ problem, selectedOption, onOptionChange }) => {
+  return (
+    <Box
+      sx={{ border: '1px solid #ddd', borderRadius: 2, p: 2, mb: 2, backgroundColor: '#f9f9f9' }}
+    >
+      <Typography variant="h6" sx={{ mb: 2 }}>
+        {problem?.question}
+      </Typography>
+      <RadioGroup value={selectedOption || ''} onChange={(event) => onOptionChange(event)}>
+        {problem?.options?.map((option, index) => (
```

Comment:
If problem.options is empty, you should consider adding a fallback UI, like displaying a message for "No options available."